src/index.ts

```diff
@@ -397,7 +396,7 @@ const plugin = {
     recommended: {
       name: '@vitest/recommended',
       plugins: {
-        get vitest(): ESLint.Plugin {
+        get vitest() {
```

Comment:
do you know why this type cause tsc failure?

Reply:
Reading from here https://typescript-eslint.io/developers/custom-rules#utils-package it is recommended to not import ESLint types, because typescript ESLint handles it already. Some Differences are described in the yellow warning boxes down. i.e. The context-retrival is different, which would in types from ESlint not include the default options.

As far as I understand, it seems a mixture of ESlint imports and Typescript ESlint imports will produce type check issues. To get it in deep, maybe asking one of the maintainers on Typescript ESlint would be necessary, as I am also just learning by reading their documentations and sources.

Reply:
i'm going to release this in the version of release. I'll see what people think about it.